src/source/mod.rs

```diff
@@ -351,6 +351,23 @@ where
     {
         blt::high_pass(self, freq)
     }
+
```

Comment:
Think you forgot the #[inline] here

```suggestion

#[inline]
```